garage/misc/gcs_utils.py

```diff
@@ -0,0 +1,97 @@
+"""Utility functions to upload files to Google Cloud Storage.
+
+Users can upload files to a specific bucket in their GCS account.
+Before attempting to upload any file, it's important to obtain an
+authentication certificate and set the corresponding environment variable
+as mentioned in section "Set up a service account" in the following link:
+    https://cloud.google.com/video-intelligence/docs/common/auth
+It's also possible to set the value of the environment variable in the field
+GCS_CREDENTIAL_PATH in file config_personal.py.
+"""
+
+import glob
+import os
+
+import google.api_core.exceptions
+from google.cloud import storage
+from termcolor import colored
+
+from garage.config import GCS_BUCKET
+from garage.config import GCS_CREDENTIAL_PATH
+from garage.config import GCS_PATH_IN_BUCKET
+
+
+def get_gcs_bucket(bucket_name: str = GCS_BUCKET):
+    """Obtain the GCS bucket.
+
+    It's verified that the JSON credentials to connect to GCS is set, and then
+    the bucket is retrieved.
+
+    Parameters
+    ----------
+    bucket_name: str
+        The bucket name as defined in GCS.
+
+    """
+    # Name of the environment variable used by the GCS library
+    gcs_env_var = "GOOGLE_APPLICATION_CREDENTIALS"
+    if gcs_env_var not in os.environ:
+        assert os.path.isfile(GCS_CREDENTIAL_PATH), colored(
+            "No credentials for GCS were provided. For instructions on " \
+            "how to setup credentials, see \"Set up a service acccount\":\n" \
+            "https://cloud.google.com/video-intelligence/docs/common"\
+            "/auth\n" \
+            "Besides an environment variable, the certificate path can " \
+            "be set in field GCS_CREDENTIAL_PATH at config_personal.py.",
+            "yellow")
+        os.environ[gcs_env_var] = GCS_CREDENTIAL_PATH
+    storage_client = storage.Client()
```

Comment:
Don't we need to specify a project?

Reply:
The project is already associated to the the GCS credential.